zeeguu/core/nlp_pipeline/__init__.py

```diff
@@ -1,8 +1,32 @@
 from .spacy_wrapper import SpacyWrapper
+from .confusion_generator import NoiseGenerator
+from .automatic_gec_tagging import AutoGECTagging
+from .confusion_set import ConfusionSet
+from os.path import join
+import json
 
 # Initialize the models, use the WV.
 SPACY_EN_MODEL = SpacyWrapper("english", False, True)
 SPACY_DK_MODEL = SpacyWrapper("danish", False, True)
 SPACY_DE_MODEL = SpacyWrapper("german", False, True)
 
-SpacyWrappers = {"en": SPACY_EN_MODEL, "dk": SPACY_DK_MODEL, "de": SPACY_DE_MODEL}
+SpacyWrappers = {"en": SPACY_EN_MODEL, "da": SPACY_DK_MODEL, "de": SPACY_DE_MODEL}
+
+# Initialize the Different components
+confusion_set_dk = {}
+with open(join("pos_dictionaries", "dk_pos_dict_w_prob_filter_f1.json"), "r") as f:
```

Comment:
Indeed, the only reason I have done it was to mainly check it was working, but I do agree it's not the best solution. 

Regarding how they are generated, I would like to make this based on the Zeeguu articles. Now that I have more time, I can try to do it in my copy Database version.

This would be what we had talked about, where we would just run the dictionary updates once a month or so. This would make the word forms more in line with what users are going to see/read based on the recommendations. As you say, these dictionaries might be useful for further research, so it could be a nice thing to have. 

Currently, the one I added is a small dictionary based on the https://gigaword.dk/ corpus, which is under the CC license, so if we use it we need to link it to the original paper. 

So, for now, maybe the best would be to have a private repository to just have this data and when we make it based on the crawling procedure it could be a separate repository that is public, what do you think?

Reply:
I think a private repository makes automating the deployment process a bit too cumbersome. Let's make it a public repo in the zeeguu org (do you have permissions to create one?). Then as long as the word lists are generated based on the gigaword corpus we add a note in the readme. And when we generate it based on the zeeguu article collection then we remove the note and that's it. What do you think?

Reply:
Sounds like a plan! 

I don't have access, so if you could create it in the way that's easiest to then deploy, I can just make a pull request with the data/references.